lambdas/enums/lambda_error.py

```diff
@@ -90,6 +93,11 @@ def create_error_body(self, params: Optional[dict] = None, **kwargs) -> str:
         "err_code": "CDR_5002",
         "message": "An error occurred when creating pre-signed url for document reference",
     }
+    CreatePatientSearchInvalid = {
+        "err_code": "CDR_5003",
+        "message": "Failed to validate patient",
+        "fhir_coding": FhirIssueCoding.INVALID,
```

Comment:
I'm not sure this is correct and contradicts with the equivelant search entry for a PDS miss maybe want to bring it with line with that?

```suggestion
        "fhir_coding": FhirIssueCoding.EXCEPTION,
```